src/app/tasks/task-viewer/task-viewer.less

```diff
@@ -71,3 +71,11 @@
     .dropdown-selector;
   }
 }
+
+
+.task-viewer-tabs {
+  li {
+    text-align:center!important;
```

Comment:
Don't use `!important` — it can lead to nasty side effects. Why can't we just add a `.text-center` class instead on the `li` element? Do we need `!important`?